README.md

```diff
@@ -38,6 +38,7 @@ docker for development if possible.
 - Navigate to `http://localhost` to access Inferno, where your test suite will
   be available. To access the FHIR resource validator, navigate to
   `http://localhost/validator`.
+- To connect to a service running locally on your machine, use `host.docker.internal` as the hostname
```

Comment:
I think you can also use the service name in `docker-compose`. Like if we have a service in docker-compose named "sarah", you could do `http://sarah:3000` and it should resolve in the docker internal network. That's what we do in the deqm test kit

Reply:
Is it possible to create a service for `measure_repository_service` that doesn't have an image but resolves to the docker internal network? I'm pretty sure we don't have an image for the measure repo service right now but an image or build context is required for a service

Reply:
Hmm no. I guess I might've misunderstood the motivation for this instruction in the first place. This is useful when:

* Running the test kit via docker
* Running the system under test locally (i.e. instead of using a publicly available deployed URL or something).

Maybe the rarest use case at this point, as I think the recommended usage if you're developing locally will be to run using foreman directly to be able to debug easier. 

Now that I think about it though, we should probably keep this test kit decoupled from our prototype, which we didn't do in deqm-test-kit (the service for deqm_test_server exists in the docker environment there). So I've changed my mind. You can ignore the first comment and keep this as is, I think it's fine